deploy/converged/crds/hco.cr.yaml

```diff
@@ -6,3 +6,5 @@ metadata:
 spec:
   CDIImagePullPolicy: IfNotPresent
   KubeVirtImagePullPolicy: IfNotPresent
+  KWebUIImagePullPolicy: IfNotPresent
```

Comment:
No longer needed.  The hco is going to have no config options

Reply:
ok, hardcoded